apps/alarm/app.js

```diff
@@ -264,7 +268,7 @@ function showEditRepeatMenu(repeat, dow, dowChangeCallback) {
     },
     /*LANG*/"Custom": {
       value: isCustom ? decodeDOW({ rp: true, dow: dow }) : false,
-      onchange: () => setTimeout(showCustomDaysMenu, 10, isCustom ? dow : EVERY_DAY, dowChangeCallback, originalRepeat, originalDow)
```

Comment:
When changing from e.g. weekday to custom
I expect only Monday to Friday to be selected,
not every day.